src/components/VariantDb/index.tsx

```diff
@@ -0,0 +1,245 @@
+import * as React from 'react';
+import { Button, Col, List, notification, Row, Typography } from 'antd';
+import { deleteCluster, getStatus, launchCluster } from './fetchVariantCluster';
+import { MAX_MINUTES_TRY, INCREMENT, clusterStatus, isInterimState } from './store';
+
+import './index.css';
+import LaunchClusterCard from './LaunchClusterCard';
+
+const { Title } = Typography;
+
+type VariantDbProps = {
+  api: Function;
+};
+
+type VariantDbState = {
+  status: string;
+  modalVisible: boolean;
+};
+
+class VariantDb extends React.Component<VariantDbProps, VariantDbState> {
+  state = {
+    status: '',
+    modalVisible: false,
+  };
+
+  data = [
+    {
+      name: 'Studies',
+      value: 13,
+    },
+    {
+      name: 'Participants',
+      value: <div>14494</div>,
+    },
+    {
+      name: 'Genes',
+      value: 21393,
+    },
+    {
+      name: 'Variants',
+      value: 29848393,
+    },
+    {
+      name: 'Exomic variants',
+      value: 2387298,
+    },
+  ];
+
+  errorNotification = (message: string, decription: string) => {
```

Comment:
I would not wrap notification... I would using directly as it is

Reply:
well i think it just adds allot of extra code. Its the same error just the massage (cause) changes.